zookeeper/values.yaml

```diff
@@ -1,7 +1,7 @@
 name: zookeeper
 image:
   repository: zookeeper
-  tag: 3.3
+  tag: 3.3.6
```

Comment:
We've actually switched to zookeeper 3.4 in monasca-docker. Do we want to use `3.4.10` instead?